convert.rb

```diff
@@ -13,6 +13,22 @@
 base = data['base']
 units = data['units']
 
+
+def get_currency_rate_with_symble(base, *targets)
+    default_base = 'EUR'
+    access_key = "5413e55dd50ddb2555c63a9cd57c0306"
+    uri = URI("http://api.exchangeratesapi.io/v1/latest?access_key=#{access_key}")
+    result = JSON.parse(Net::HTTP.get(uri))
+    res = Hash['success' => result['success'], 'base' => base, 'timestamp' => result['timestamp']]
+    rates = Hash[]
+    targets.each do | target |
+        rates[target] = result['rates'][target] * result['rates'][default_base] / result['rates'][base]
+    end
+    res['rates'] = rates
+    return res
+end
```

Comment:
I guess this is because the free plan for exchangeratesapi.io always uses EUR as the base currency.